src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.starts {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+&--5 .stars__star,
+&--1 .stars__star:nth-child(1),
+&--2 .stars__star:nth-child(-n + 2),
+&--3 .stars__star:nth-child(-n + 3),
+&--4 .stars__star:nth-child(-n + 4) {
```

Comment:
The use of BEM modifiers (`&--5`, `&--1`, etc.) should be consistent with the BEM naming convention. Ensure that these modifiers are applied to the correct base class (`.stars`) and are used consistently throughout the project.